app/views/ops/_rbac_group_details.html.haml

```diff
@@ -1,6 +1,9 @@
 - if @edit
   - url = url_for_only_path(:action => 'rbac_group_field_changed', :id => (@edit[:group_id] || "new"))
   - combo_url = "/ops/rbac_group_field_changed/#{@edit[:group_id] || 'new'}"
+- amazon_authentication = ::Settings.authentication.mode.casecmp('amazon') == 0
+- database_authentication = ::Settings.authentication.mode.casecmp('database') == 0
+- ext_auth_with_saml = ::Settings.authentication.mode.casecmp("httpd") == 0 && ::Settings.authentication.saml_enabled
```

Comment:
@ZitaNemeckova Thank you for putting this together. 

I have a suggestion that might simplify the code.

Group lookup is only supported when:

1. mode is either ldap or ldaps
2. mode is httpd and saml is **not** enabled

So maybe something like this ruby-ish pseudo code would be better:

```ruby
mode = ::Settings.authentication.mode
saml_enabled = ::Settings.authentication.saml_enabled
can_lookup_groups = false
can_lookup_groups = mode.downcase.start_with?('ldap')
can_lookup_groups = mode.downcase == "httpd" && ! saml_enabled unless can_lookup_groups
```

Then the check below changes from:
```diff
- if !amazon_authentication && !database_authentication && !ext_auth_with_saml
+ if can_lookup_groups
```

Hope this helps! JoeV

Reply:
@jvlcek Thanks, looks way better 👍